### ISSUE TITLE
For zh-CN users, defaults to FxA/Sync services hosted in China

### ISSUE THREAD

**Issue by l-hedgehog** [mozilla-lockwise/lockwise-ios#460]
... and provide a switch so one can still choose the global services.

Similar to what was done to Firefox for iOS in mozilla-mobile/firefox-ios#1312

The majority of Firefox users in China are using builds distributed by Beijing office, and we run our own FxA/Sync services to support these builds, since global services on AWS are not stable for ordinary Chinese users. If you have more questions about this request, please reach out, my email is <bzhao@...>. Thanks!

**Comment by joeyg** [comment]
@sandysage Any idea of what the flow should be for CN users? The linked PR checks the user's locale and if it is zh_CN then provides the option to user the alternate sync settings. Would that option make sense on the welcome screen?

**Comment by sandysage** [comment]
@l-hedgehog this is really helpful and insightful! I didn't realize this is how we support Chinese FxA/Sync endpoints in Firefox for iOS. This is great context for us for when we start branching out.

Right now Chinese/China is not on our roadmap for the near term. We'll need to pick this back up when we are ready to go to this market specifically. I'm leaving this in our Inbox for future consideration.

@joeyg thanks for checking in here. I'm pretty sure @changecourse will have more to offer for the desired flows when we are focused on this market.

**Comment by l-hedgehog** [comment]
Hi @sandysage, thanks for the response.

> Right now Chinese/China is not on our roadmap for the near term. We'll need to pick this back up when we are ready to go to this market specifically. I'm leaving this in our Inbox for future consideration.

IIUC this means Lockbox will only be available in App Store of selected regions (not including China) when it's first released?

**Comment by sandysage** [comment]
> IIUC this means Lockbox will only be available in App Store of selected regions (not including China) when it's first released?

@l-hedgehog that is correct. We're specifically targeting a handful of regions, as noted in [this comment](https://github.com/mozilla-lockbox/lockbox-ios/issues/176#issuecomment-390737930), to keep our initial launch limited to minimize risk and focus on learning from a small group of users to start.

**Comment by l-hedgehog** [comment]
FYI, with the deployment of recently rebased server codes earlier today, it's now possible to authenticate with FxA/Sync services in China with the OAuth login flow. I can dump the synced passwords with the script at rfk/boxlocker, after changing ISSUER to accounts.firefox.com.cn and nothing else.
